Tensile/Utilities/merge.py

```diff
@@ -147,8 +149,44 @@ def debug(*args, **kwargs):
     if verbosity < 2: return
     msg(*args, **kwargs)
 
+# Tags distinguishing solution types
+# Can be added to size key to allow solutions of each type to be present
+# in logic file for a given size
+class SolutionTag(IntEnum):
+    VALU = 0
+    MFMA = 1
+
+    def __str__(self):
+        return ["VALU", "MFMA"][self]
+    def __repr__(self):
+        return str(self)
+
+def getSolutionTag(solution):
+    if solution["EnableMatrixInstruction"] or solution["MatrixInstruction"]:
```

Comment:
CI reports failure on this line because you are querying a nonexistent key. You should check that if key exists in the first place, or catch `KeyError` exception.

Reply:
Odd, that was working on my branch. I'll take a look into this.

Reply:
Looks like I made a change after running the tests that broke them, should be good now!